### PR TITLE
Media 2021 queries - video

### PR DESCRIPTION
**Pull request by dougsillars**
Hi - here are my initial queries for the video section of the media chapter

Makes progress on #2144

### CODE REVIEW

File: sql/2021/media/video_attribute_count_desktop.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    url AS pageURL,
+    JSON_VALUE(payload, "$._media") AS media,
+    CAST(JSON_VALUE(JSON_VALUE(payload, "$._media"), "$.num_video_nodes") AS INT64) AS num_video_nodes,
+    (JSON_QUERY_ARRAY(JSON_VALUE(payload, "$._media"), "$.video_durations")) AS video_duration,
+    (JSON_QUERY_ARRAY(JSON_VALUE(payload, "$._media"), "$.video_display_style")) AS video_display_style,
+    ARRAY_TO_STRING(JSON_QUERY_ARRAY(JSON_VALUE(payload, "$._media"), "$.video_attributes_values_counts"), " ") AS video_attributes_values_counts,
+    (JSON_QUERY_ARRAY(JSON_VALUE(payload, "$._media"), "$.video_source_format_count")) AS video_source_format_count,
+    (JSON_QUERY_ARRAY(JSON_VALUE(payload, "$._media"), "$.video_source_format_type")) AS video_source_format_type
+  FROM
+    `httparchive.summary_pages.2021_07_01_desktop`
```
[the remark this answers is not in the document]
Comment (dougsillars) [reply sql/2021/media/video_attribute_count_desktop.sql]: My priority this week was getting my draft completed.  I'll get to this next week :)

[the remark this answers is not in the document]
Comment (rviscomi) [reply sql/2021/media/video_attribute_count_desktop.sql]: Oh I hadn't read the queries/filenames closely enough, but now I see you do have desktop and mobile data for (almost) all of the queries—they're just split into pairs of SQL files. So you are able to write the draft with the full context of the results and this is more of a formatting nit. Carry on! :)

Just be aware that `vide_ext_desktop.sql` has no mobile equivalent.

[the remark this answers is not in the document]
Comment (dougsillars) [reply sql/2021/media/video_attribute_count_desktop.sql]: yeah - I can make them all into one with a little effort, but if you're ok with this approach, so am I :)
I have all the data, and my draft is complete.  Updated the last one to be mobile & desktop.

**Comment by rviscomi** [comment]
@dougsillars could you take a look at the [linter errors](https://github.com/HTTPArchive/almanac.httparchive.org/runs/4086855435?check_suite_focus=true)? Let me know when that and the remaining comments are resolved and we'll get this merged ASAP.

**Comment by dougsillars** [comment]
@rviscomi The linter is no longer mad at me :)

**Pull request merged**